### PR TITLE
Fix: enable UART console on Joule

### PR DESCRIPTION
**Pull request by SevenQC**
Since kernel v4.4, UART driver is refactored as a subdevice of LPSS.
And the kernel configs related to LPSS and 8250_dw have to be set as Y
to enable the UART console.

Jira: None
Test: Serial logs come out on Joule when add kernel cmdline console=/dev/ttyS2.

Signed-off-by: Qin Chao <chao.qin@intel.com>

### CODE REVIEW

File: kernel_config/kernel_64_defconfig [hunk 1]
```diff
@@ -2815,7 +2815,7 @@ CONFIG_SERIAL_8250_SHARE_IRQ=y
 CONFIG_SERIAL_8250_DETECT_IRQ=y
 CONFIG_SERIAL_8250_RSA=y
 # CONFIG_SERIAL_8250_FSL is not set
-CONFIG_SERIAL_8250_DW=m
+CONFIG_SERIAL_8250_DW=y
```
Comment (kalyankondapally) [inline kernel_config/kernel_64_defconfig]: Can u make u commit message more verbose. Add why u need this, Jira and what should we test ?

**Comment by kalyankondapally** [comment]
Thanks Jira and Test are still missing

**Comment by kalyankondapally** [comment]
@shobhitka ping can u review this ?

**Comment by SevenQC** [comment]
Sorry for the misoperation on this pull request. And I will close this and create new one.

**Comment by kalyankondapally** [comment]
@shobhitka FIne with this now to be merged ?

**Comment by kalyankondapally** [comment]
@SevenQC Can u rebase the patch ?

**Comment by shobhitka** [comment]
Looks like more commits are added in this pull request. We should remove the merge commit and I see UART commit also two times with no Jira/Test detail update. maybe I am not able to refer correct pull request ?

Also why we need new fstab for squashf. During installation we are making it as ext4 so that we can easily remount as rw if needed. So squahsfs is uses a is only in case of live boot which does not rely on fstab anyway and does loopback mount of squashfs image file.

**Comment by kalyankondapally** [comment]
merged squashfs pull request is in mixin side, lets continue conversation there.

**Pull request closed without merging**